docs/throttling.md

```diff
@@ -27,15 +27,14 @@ Lighthouse, by default, uses simulated throttling as it provides both quick eval
 
 ## DevTools' Lighthouse Panel Throttling
 
-In Chrome 79 and earlier, you could choose between [the throttling types](#types-of-throttling) of Simulated, Applied, and none.
+The Lighthouse panel has a simplified throttling setup:
 
-Starting with Chrome 80, the Audits panel is simplifying the throttling configuration:
+1. _Simulated throttling_ remains the default setting. This matches the setup of PageSpeed Insights and the Lighthouse CLI default, maintaining cross-tool consistency.
+   - If you click the `View Original Trace` button, the trace values will not match up with Lighthouse's metric results, as the original trace is prior to the simulation.
```

Comment:
I know I'm a broken record on this but I thought we had decided to remove the View Trace button in this state?

![emperorsnewgroove](https://user-images.githubusercontent.com/316891/146277245-ff244630-46f2-4b09-8f8e-0411318b11a3.gif)